frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -28,7 +28,7 @@
 
         component(
           :is='messageType(message)'
-          :key='message.id'
+          :key='message.id + Math.random()'
```

Comment:
This change is related to the "duplicate keys" bug of #1227, but a more correct fix should be used. /cc @Silver-IT

Reply:
Make sense.